web/src/features/charts/elements/AreaGraphLayers.tsx

```diff
@@ -53,7 +53,7 @@ function AreaGraphLayers({
     .x((d: any) => timeScale(d.data.datetime))
     .y0((d) => valueScale(d[0]))
     .y1((d) => valueScale(d[1]))
-    .defined((d) => (shouldHideEmptyData ? d[1] > 0 : Number.isFinite(d[1])));
+    .defined((d) => Number.isFinite(d[1]));
 
   // Mouse hover events
   let mouseOutTimeout: string | number | NodeJS.Timeout | undefined;
```

Comment:
Unrelated but why does this have a NodeJS.Timeout type when it's only running in the browser?

Reply:
No idea :(